packages/early-boot-config/early-boot-config.spec

```
@@ -0,0 +1,152 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}early-boot-config
+Version: 0.0
+Release: 0%{?dist}
+Summary: early-boot-config
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source100: early-boot-config.service
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package -n %{_cross_os}early-boot-config-common
+Summary: early-boot-config binary
+%description -n %{_cross_os}early-boot-config-common
+%{summary}.
```

nit: I like to separate the description from the definition:
```suggestion
%package -n %{_cross_os}early-boot-config-common
Summary: early-boot-config binary

%description -n %{_cross_os}early-boot-config-common
%{summary}.
```

(Essentially my style rule is "have an empty line before any of the built-in rpm sections".)